.travis.yml

```diff
@@ -33,10 +32,18 @@ jobs:
         - yarn lint
         - yarn test
     - stage: deploy
-      name: 'Release recomponents to NPM'
+      name: 'Release recomponents to NPM and deploy storybook to github pages'
       before_script:
         - cd packages/recomponents
       if: branch = master
```

Comment:
@followyouhome Just curious why I pushed a PR, the script is executed? so what does `if: branch = master` mean here?

Reply:
to prevent duplicated builds - w/o this condition we will have build for each branch + build for PR

Reply:
Agreed. With this condition, it looks like when PR is merged to master, the deploy will be executed, right? But I noticed when I created a PR, it is executed as well...

Reply:
are you sure it was the deploy script that executed? This file is still configured to `build`, `lint`, and `test` on all PRs

Reply:
Let's first agree with below point?
1) if master: build, deploy storybook, semantic-release
2) if other branches(PR): lint, test
Then let's adjust the `.travis.yml` file.